### ISSUE TITLE
Sequencer constantly logs "Bid Post Error"

### ISSUE THREAD

**Issue by rhlsthrm** [connext/nxtp#1455]
**Describe the bug**
Logs are filled with this error.

**To Reproduce**
See log filter here:

https://app.datadoghq.com/logs?query=-service%3A%28lighthouse%20OR%20router%29%20-container_name%3Atestnet-staging-sequencer&cols=host%2Cservice%2C%40error.context.transferId&index=core-services-testnet&integration_id=&integration_short_name=&messageDisplay=inline&saved_view=1018000&stream_sort=desc&viz=stream&from_ts=1656430364735&to_ts=1656431264735&live=true

**Additional context**
Seeing this log a lot: `MissingXCall: No XCall was found in the subgraph for this auction.` This should never happen so we need to track down why routers are sending a bid which doesn't exist on the sequencer's subgraph.

Fixes: #1679

**Comment by jakekidd** [comment]
> why routers are sending a bid which doesn't exist on the sequencer's subgraph.

Double check that the sequencer's subgraph querying is correct/up-to-date

**Comment by jakekidd** [comment]
Also, does the sequencer use a maxBlockNumber for its subgraph queries? If it uses maxBlockNumber and sets it to like, 25 blocks ago, that might explain why we are having this issue

**Comment by jakekidd** [comment]
Nvm, looks like no usage of maxBlockNumber,
```ts
    transfer = await subgraph.getOriginTransferById(origin, transferId);
```

**Comment by sanchaymittal** [comment]
should be fixed by #1679
